core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -144,4 +146,72 @@ public static Snapshot snapshotAfter(Table table, long snapshotId) {
     throw new IllegalStateException(
         String.format("Cannot find snapshot after %s: not an ancestor of table's current snapshot", snapshotId));
   }
+
+  /**
+   * Returns the ID of the most recent snapshot for the table as of the timestamp.
+   *
+   * @param table a {@link Table}
+   * @param timestampMillis the timestamp in millis since the epoch
```

Comment:
When mentioning epoch, I think it should state _which_ epoch. I'd update this to "Unix epoch"

Reply:
Ok